weed/mount/weedfs_file_mkrm.go

```diff
@@ -110,6 +111,8 @@ func (wfs *WFS) Mknod(cancel <-chan struct{}, in *fuse.MknodIn, name string, out
 
 }
 
+var swapFileNameRegexp = regexp.MustCompile(`^\.([a-zA-Z0-9._-]+)\.swp$`)
```

Comment:
Not generic.

Reply:
Do you have any suggestion?

Reply:
Just document the consequences when worm is enabled? Then if we have a solution to delete the file (swap or whatever), provide that solution together in the document. 

If above are provided, we can delete the speicial hanling here?

Reply:
removed. It seams that vi will create temp files not only .swp, but also .swx (x = [a-z]), and other names.
We should document that vi (and other editiors which will create temp files) is not allowed in WORM mode